Sync the content of sso72-dev and sso72-cp branches to sso72 branch post RH-SSO 7.2.1.CE_Sprint#16 and RH-SSO 7.2.2.GA images release

iankko opened the pull request:
Sync the content of ```sso72-dev``` and ```sso72-cp``` branches into ```sso72``` branch post the release of:

- RH-SSO 7.2.1.CE_Sprint#16 and
- RH-SSO 7.2.2.GA

images. Perform this by cherrypicking the following changes into the ```sso72``` branch:

1. ```sso72-dev``` to ```sso72``` branches sync:

    - [Update to RH-SSO 7.2.1.CR3.](https://github.com/jboss-container-images/redhat-sso-7-image/commit/f556c169914597cd09eebe791ebcab7c0c0582c6)

2. ```sso72-cp``` to ```sso72``` branches sync:

    - [Update to RH-SSO 7.2.2.CR1 ](https://github.com/jboss-container-images/redhat-sso-7-image/commit/561140d1187d3cb4297945c3add58ee9eafbc44e)
    - [Backport of: [CLOUD-2347] Container namespace switch change to sso72-cp branch](https://github.com/jboss-container-images/redhat-sso-7-image/commit/68b678e10c39573f9e38029ec455e2892e115e91)


       Note: The commit message of this change has been amended to say ```sso72``` branch (instead of ```sso72-cp``` like in the original)

The rest of the changes / commits done either in ```sso72-dev``` branch, or ```sso72-cp``` branch are already present in ```sso72``` branch too.


Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

iankko commented:
@goldmann @ASzc @rcernich PTAL

rcernich commented on image.yaml:
```diff
@@ -3,8 +3,10 @@ schema_version: 1
 description: "Red Hat Single Sign-On 7.2 container image"
 version: "7.2.0"
-from: "jboss-eap-7/eap71:7.1.0-9"
+from: "jboss-eap-7/eap71:latest"
```
@goldmann, do we want to stick with latest, or should we pin this to the latest released version, e.g. 7.1.1, to ensure we get the latest released version and not just the latest version.

iankko replied:
@goldmann Hey Marek, any hint on the above question? (we need to have this one solved && merged before #38 one can be merged yet).

Thanks, Jan

P.S.: Ignore the merge conflicts below. Will address them, but would want to know the conclusion on this before doing that.

goldmann replied:
You can pin to latest released image (check with EAP team which one is it) to be sure which image you use. By using `latest` you have no control over which image you pull now. This will change in the future slight;y.

ASzc committed:
Update to RH-SSO 7.2.2.CR1

Commit:
Backport of:

  [CLOUD-2347] Container namespace switch

change to sso72 branch

Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

iankko commented:
@ASzc @goldmann @rcernich

Updated the PR to:

- Fix the previous merge conflicts
- Also added Alex's change for RH-SSO 7.2.2.GA image release from sso72-cp branch (so sso72 one is synced with this change too)

PTAL

iankko commented:
@rcernich Thanks for the review, Rob!
